src/library/Uno.Material/Uno.Material.csproj

```diff
@@ -51,34 +51,34 @@
 
 			<PropertyGroup>
 				<DefineConstants>$(DefineConstants);WinUI</DefineConstants>
-				<DefineConstants Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'">$(DefineConstants);WinUI_Desktop</DefineConstants>
+				<DefineConstants Condition="$(TargetFramework.Contains('-windows10'))">$(DefineConstants);WinUI_Desktop</DefineConstants>
 				<PackageId>Uno.Material.WinUI</PackageId>
-				<TargetFrameworks Condition="'$(TargetFrameworkOverride)'=='' and '$(OS)'=='Windows_NT'">$(TargetFrameworks);net5.0-windows10.0.18362</TargetFrameworks>
+				<TargetFrameworks Condition="'$(TargetFrameworkOverride)'=='' and '$(OS)'=='Windows_NT'">$(TargetFrameworks);net6.0-windows10.0.18362</TargetFrameworks>
 			</PropertyGroup>
 
 			<ItemGroup>
 				<PackageReference Include="Uno.WinUI" Version="4.5.12" />
 
-				<PackageReference Include="Uno.WinUI.Lottie" Version="4.5.12" Condition="'$(TargetFramework)'!='net5.0-windows10.0.18362'" />
+				<PackageReference Include="Uno.WinUI.Lottie" Version="4.5.12" Condition="!$(TargetFramework.Contains('-windows10'))" />
 
-				<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.0" Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'" />
+				<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.0" Condition="$(TargetFramework.Contains('-windows10'))" />
 
-				<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22000.24" Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'" />
-				<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22000.24" Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'" />
+				<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22000.24" Condition="$(TargetFramework.Contains('-windows10'))" />
+				<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22000.24" Condition="$(TargetFramework.Contains('-windows10'))" />
 			</ItemGroup>
 
-			<ItemGroup Condition="'$(TargetFramework)'=='net5.0-windows10.0.18362'">
+			<ItemGroup Condition="$(TargetFramework.Contains('-windows10'))">
```

Comment:
Given the number of `$(TargetFramework.Contains('-windows10'))`, you can probably create a single variable at the top:
```xml
<PropertyGroup>
<_IsWindows>false</_IsWindows>
<_IsWindows $(TargetFramework.Contains('-windows10'))>true</_IsWindows>
</PropertyGroup>
```

Then use this elsewhere:
```xml
<ItemGroup Condition="$(_IsWindows )">
```

Reply:
Yep, I'll bring over the same thing we do for Toolkit, the [PlatformItemGroups.props](https://github.com/unoplatform/uno.toolkit.ui/blob/main/src/PlatformItemGroups.props)

Reply:
@kazo0 you might want to update the PlatformItemGroups.props to make windows detection more flexible - it's looking for net6.0-windows, so change to -windows so as we move to net7 it won't break